core/server/worker/src/main/java/alluxio/worker/block/UnderFileSystemBlockReader.java

```diff
@@ -293,9 +300,9 @@ private void updateUnderFileSystemInputStream(long offset) throws IOException {
     }
 
     if (mUnderFileSystemInputStream == null && offset < mBlockMeta.getBlockSize()) {
-      UfsInfo ufsInfo = mUfsManager.get(mBlockMeta.getMountId());
-      UnderFileSystem ufs = ufsInfo.getUfs();
-      mUfsMountPointUri = ufsInfo.getUfsMountPointUri();
+      UfsManager.UfsClient ufsClient = mUfsManager.get(mBlockMeta.getMountId());
```

Comment:
Is there any chance that this `ufsClient` is not the same as the `ufsClient` we used to get the `UFSResource`?